### PR TITLE
style: Fixed Misspells

### LINKED ISSUE #198: Fix misspell
### Describe the issue you're facing

There are few spelling mistakes in paralus as reported in Go report card: https://goreportcard.com/report/github.com/paralus/paralus#misspell

Screenshot from 2023-04-04 21-05-32

Let's fix them up.

### PR DESCRIPTION
**Pull request by Kartik-Garg**
### What does this PR change?

-Fixed the various misspells which were present in the code pointed out by Go Report Card Fixes #198

### Does the PR depend on any other PRs or Issues? If yes, please list them.

-No

### Checklist

I confirm, that I have...

- [x] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Added tests for this PR
- [ ] Formatted the code using `go fmt` (if applicable)
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [x] Updated `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md
Comment (niravparikh05) [inline CHANGELOG.md:14]: this should not be under 0.2.2 release section, can you move it up. or run make changelog to autopopulate this.

**Review by niravparikh05 — COMMENTED** [review]
just a tiny nit, looks good otherwise.

File: CHANGELOG.md
Comment (akshay196) [reply CHANGELOG.md:14]: No need to update changelog in PRs. Changelog can be automatically generated later when we run `make changelog` in main branch.

Comment (akshay196) [reply CHANGELOG.md:14]: Just following the [commit message guidelines](https://github.com/paralus/paralus/blob/main/CONTRIBUTING.md#commit-message-guidelines) is enough.

Comment (Kartik-Garg) [reply CHANGELOG.md:14]: Alright, I will just remove changes from changelog then

**Comment by Kartik-Garg** [comment]
> just a tiny nit, looks good otherwise.

Thanks for the comments, I have removed the changes from changelog @niravparikh05

**Review by akshay196 — APPROVED** [review]

**Comment by akshay196** [comment]
@Kartik-Garg This looks good. Only one change: The commit message subject should be all small letters as per the guidelines. 
It's OK for now, we can change the message while merging this PR.

Waiting for @niravparikh05's approval before merging this.

**Review by niravparikh05 — APPROVED** [review]
lgtm

**Review by niravparikh05 — APPROVED** [review]
lgtm

**Pull request merged**
